src/components/Toggle/Toggle.scss

```diff
@@ -76,36 +79,59 @@
 
 // Slider mixin
 @mixin ms-Toggle-slider($direction) {
-  // Slider pseudo element
+  // Slider background element
   &:before {
+    content: '';
+    width: 0;
+    position: absolute;
+    height: 16px;
+    top: 0;
+    left: 0;
+    background-color: $ms-color-themeDark;
+    outline: 2px solid transparent;
+    transition: width 0.3s;
```

Comment:
Could we increase the speed to 0.167s and use one of the easing curves from our [current animation classes](http://dev.office.com/fabric/styles#animations)?